src/Redist/Microsoft.ML.TensorFlow.Redist/Microsoft.ML.TensorFlow.Redist.proj

```diff
@@ -9,20 +9,12 @@
     <TensorFlowConfig Include="windows" FileExtension=".zip" FilesFromArchive="lib\tensorflow.dll;include\tensorflow\c\LICENSE" Runtime="win-x64"/>
```

Comment:
nit: could you format the windows config the same way you did below for linux and mac?
Not important if you are not doing another iteration.